tokio/tests/io_async_fd.rs

```diff
@@ -825,7 +825,7 @@ async fn await_error_readiness_invalid_address() {
         msg.msg_iovlen = 1;
 
         if unsafe { libc::sendmsg(socket_fd, &msg, 0) } == -1 {
-            Err(std::io::Error::last_os_error()).unwrap()
+            panic!("{:?}", std::io::Error::last_os_error())
```

Comment:
Same here.

Reply:
It's test only code and there is already the same panic! in the same file so I think it's fine as-is.

https://github.com/tokio-rs/tokio/blob/16eb7e3f9427c7070899f62c24fd79dcf29b1a03/tokio/tests/io_async_fd.rs#L791

Reply:
Ah, you're right. I don't care about tests.